$MirOS: ports/www/firesomething/patches/patch-browser_installer_unix_packages-static,v 1.1.2.1 2008/12/02 19:40:50 bsiegert Exp $
--- browser/installer/unix/packages-static.orig	Wed Aug 27 01:56:24 2008
+++ browser/installer/unix/packages-static	Tue Dec  2 19:03:02 2008
@@ -30,15 +30,15 @@ bin/updater.ini
 bin/dictionaries/*
 
 [xpcom]
-bin/libmozjs.so
-bin/libplc4.so
-bin/libplds4.so
-bin/libxpcom.so
-bin/libxpcom_core.so
-bin/libnspr4.so
-bin/libxul.so
-bin/components/libxpinstall.so
-bin/components/libjar50.so
+bin/libmozjs.so._SO_VERSION_
+bin/libplc4.so._SO_VERSION_
+bin/libplds4.so._SO_VERSION_
+bin/libxpcom.so._SO_VERSION_
+bin/libxpcom_core.so._SO_VERSION_
+bin/libnspr4.so._SO_VERSION_
+bin/libxul.so._SO_VERSION_
+bin/components/libxpinstall.so._SO_VERSION_
+bin/components/libjar50.so._SO_VERSION_
 
 [browser]
 ; [Base Browser Files]
@@ -48,10 +48,10 @@ bin/application.ini
 bin/platform.ini
 bin/mozilla-xremote-client
 bin/run-mozilla.sh
-bin/plugins/libnullplugin.so
-bin/libsqlite3.so
+bin/plugins/libnullplugin.so._SO_VERSION_
+bin/libsqlite3.so._SO_VERSION_
 bin/README.txt
-bin/libjemalloc.so
+bin/libjemalloc.so._SO_VERSION_
 bin/blocklist.xml
 
 ; [Components]
@@ -118,8 +118,8 @@ bin/components/imglib2.xpt
 bin/components/intl.xpt
 bin/components/jar.xpt
 bin/components/jsconsole.xpt
-bin/components/libjsd.so
-bin/components/libimgicon.so
+bin/components/libjsd.so._SO_VERSION_
+bin/components/libimgicon.so._SO_VERSION_
 bin/components/jsdservice.xpt
 bin/components/layout_base.xpt
 bin/components/layout_printing.xpt
@@ -232,8 +232,8 @@ bin/components/nsSessionStore.js
 bin/components/sessionstore.xpt
 bin/components/nsURLFormatter.js
 bin/components/urlformatter.xpt
-bin/components/libbrowserdirprovider.so
-bin/components/libbrowsercomps.so
+bin/components/libbrowserdirprovider.so._SO_VERSION_
+bin/components/libbrowsercomps.so._SO_VERSION_
 bin/components/txEXSLTRegExFunctions.js
 bin/components/nsLivemarkService.js
 bin/components/nsTaggingService.js
@@ -242,7 +242,7 @@ bin/components/nsContentPrefService.js
 bin/components/nsContentDispatchChooser.js
 bin/components/nsHandlerService.js
 bin/components/nsWebHandlerApp.js
-bin/components/libdbusservice.so
+bin/components/libdbusservice.so._SO_VERSION_
 bin/components/aboutRobots.js
 bin/components/nsBadCertHandler.js
 
@@ -256,10 +256,10 @@ bin/components/nsUrlClassifierLib.js
 bin/components/url-classifier.xpt
 
 ; Kerberos NegotiateAuth
-bin/components/libauth.so
+bin/components/libauth.so._SO_VERSION_
 
 ; GNOME hooks
-bin/components/libmozgnome.so
+bin/components/libmozgnome.so._SO_VERSION_
 bin/components/mozgnome.xpt
 
 ; [Browser Chrome Files]
@@ -346,30 +346,30 @@ bin/components/dom_svg.xpt
 
 ; [Personal Security Manager]
 ;
-bin/libnssckbi.so
+bin/libnssckbi.so._SO_VERSION_
 bin/components/pipboot.xpt
 bin/components/pipnss.xpt
 bin/components/pippki.xpt
-bin/libnss3.so
-bin/libnssutil3.so
-bin/libsmime3.so
+bin/libnss3.so._SO_VERSION_
+bin/libnssutil3.so._SO_VERSION_
+bin/libsmime3.so._SO_VERSION_
 bin/libsoftokn3.chk
-bin/libsoftokn3.so
+bin/libsoftokn3.so._SO_VERSION_
 bin/libfreebl3.chk
-bin/libfreebl3.so
-bin/libssl3.so
-bin/libnssdbm3.so
+bin/libfreebl3.so._SO_VERSION_
+bin/libssl3.so._SO_VERSION_
+bin/libnssdbm3.so._SO_VERSION_
 bin/chrome/pipnss.jar
 bin/chrome/pippki.jar
 bin/chrome/pippki.manifest
 
 ; for Solaris SPARC
 bin/libfreebl_32fpu_3.chk
-bin/libfreebl_32fpu_3.so
+bin/libfreebl_32fpu_3.so._SO_VERSION_
 bin/libfreebl_32int_3.chk
-bin/libfreebl_32int_3.so
+bin/libfreebl_32int_3.so._SO_VERSION_
 bin/libfreebl_32int64_3.chk
-bin/libfreebl_32int64_3.so
+bin/libfreebl_32int64_3.so._SO_VERSION_
 
 ; [Updater]
 ;
@@ -384,8 +384,8 @@ bin/Throbber-small.gif
 
 ; [Extensions]
 ;
-bin/components/libnkgnomevfs.so
-bin/components/libauth.so
+bin/components/libnkgnomevfs.so._SO_VERSION_
+bin/components/libauth.so._SO_VERSION_
 
 ; [Additional Developer Tools]
 [adt]
